sql/core/src/test/resources/sql-tests/results/group-analytics.sql.out

```diff
@@ -223,12 +223,19 @@ grouping_id() can only be used with GroupingSets/Cube/Rollup;
 
 
 -- !query 16
-SELECT course, year, grouping__id FROM courseSales GROUP BY CUBE(course, year)
+SELECT course, year, grouping__id FROM courseSales GROUP BY CUBE(course, year) ORDER BY grouping__id, course, year
 -- !query 16 schema
-struct<>
+struct<course:string,year:int,grouping__id:int>
 -- !query 16 output
-org.apache.spark.sql.AnalysisException
-grouping__id is deprecated; use grouping_id() instead;
+Java    2012    0
```

Comment:
Are you manually editing this `group-analytics.sql.out`? The test failure is due to mismatching between spaces and tab.